src/Core/Infrastructure/RealTime/Repository/Acknowledgement/DbReadAcknowledgementRepository.php

```diff
@@ -46,6 +46,14 @@ public function findOnGoingAcknowledgementByHostId(int $hostId): ?Acknowledgemen
         return $this->findOnGoingAcknowledegementRequest($hostId, 0);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function findOnGoingAcknowledgementByHostIdAndServiceId(int $hostId, int $serviceId): ?Acknowledgement
+    {
+        return null;
+    }
+
```

Comment:
you can mock this return in your test, so it could be not necessary to implement this method in this PR.